EIPS/eip-4671.md

```diff
@@ -0,0 +1,468 @@
+---
+eip: 4671
+title: Non-Tradable Tokens
+description: A standard interface for non-tradable tokens.
+author: Omar Aflak (@omaraflak), Pol-Malo Le Bris, Marvin Martin (@MarvinMartin24)
+discussions-to: https://ethereum-magicians.org/t/eip-4671-non-tradable-token/7976?u=omaraflak
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-01-13
+requires: 165
+---
+
+# Non-Tradable Token Standard
```

Comment:
```suggestion
```
The EIP should only have sections mentioned in [EIP-1](./eip-1.md), so please remove this